openlibrary/templates/lib/nav_head.html

```diff
@@ -1,8 +1,14 @@
 $def with (page)
+$ is_privileged_user = ctx.user and (ctx.user.is_admin() or ctx.user.is_usergroup_member('/usergroup/super-librarians'))
 
 <header id="header-bar" class="header-bar">
-  $if ctx.user:
+  $if is_privileged_user:
     $ loginLinks = [
+    $     { "href": homepath() + "/merges", "text": _("Pending Requests"),"track": "MainNav|MyRequests" },
+    $ ]
+
+  $if ctx.user:
+    $ loginLinks += [
```

Comment:
When logged in as a non-privileged user, the navbar is failing to render.  `loginLinks` does not exist for this case, so the `+=` operator will not work.
Suggest moving the `is_privileged_user` check below the `loginLinks` declaration, and insert the `/merges` link into the list if `True`.